docs/dev/cli/virtual-usb/start-server.md

```diff
@@ -4,67 +4,129 @@ title: "Virtual USB CLI: Start the Server"
 sidebar_label: Start Server
 ---
 
-The [`server`](/mobile-apps/features/virtual-usb#start-server) command connects your local machine to the Sauce Labs Data Center where your Real Device Cloud private devices are hosted. You must start this command in its own separate command line terminal, since it acts as a log running continuously.
+## Description
 
-## Required
+[Connect your local machine to the Sauce Labs Data Center](/mobile-apps/features/virtual-usb#start-server) where your Real Device Cloud private devices are hosted. You must initiate this command in its own separate command line terminal, since it acts as a continuously running log.
 
----
-### `--datacenter`
-__Description__: defines the Data Center where your Real Device Cloud private devices are hosted. Possible values: `EU` or `US`.
+## Usage
 
-```java title="Basic Example (required flags only)"
-java -jar virtual-usb-client.jar server --datacenter US
-```
+<span className="cli">$ &lt;main class&gt; [OPTIONS] server [OPTIONS]</span>
 
-```java title="Sample Response"
-19:06:10.060 [main] INFO com.saucelabs.vusb.client.Runner - Runner Version 2.0.0
-19:06:11.032 [main] INFO com.saucelabs.vusb.client.server.VirtualUsbServer - Initializing vUSB-Server...
-19:06:11.038 [main] INFO com.saucelabs.vusb.client.server.VirtualUsbServer - vUSB-Server initialized
-19:06:11.039 [main] INFO com.saucelabs.vusb.client.server.rest.WebServer - Starting Virtual USB server on port 33657.
-19:06:11.103 [main] INFO com.saucelabs.vusb.client.server.rest.WebServer - Virtual USB server is up.
-```
 
-## Optional
+## Options Details
 
-These flags provide additional configuration options.
+### <span className="cli">--datacenter</span>
 
----
-### `--serverHost`
-__Description__: specifies a Virtual USB server host. Default value: `http://127.0.0.1`.
+<div className="cli-desc">
+<p><small>| REQUIRED | ENUM |</small></p>
 
----
-### `--serverPort`
-__Description__: specifies a Virtual USB server port. Default value: `33657`.
+Specifies the Sauce Labs data center where your Real Device Cloud private devices are hosted. Valid values are `EU` or `US`.
 
----
-### `--adbPortMin`
-__Description__: specifies a Virtual USB server Android Debug Bridge (ADB) port. Default value: `7000`.
+</div>
 
----
-### `--adbPortRange`
-__Description__: specifies a Virtual USB server ADB port range. Default value: `100`.
+### <span className="cli">--serverHost</span>
 
----
-### `--proxyHost`
-__Description__: specifies a HTTP proxy host.
+<div className="cli-desc">
+<p><small>| OPTIONAL | URL ADDRESS |</small></p>
 
----
-### `--proxyPort`
-__Description__: specifies a HTTP proxy port. Default value: `0`.
+A specific Virtual USB server host address. The default value, if not specified, is `http://127.0.0.1`.
```

Comment:
Hi @sweeneyskirt-sl -- this description and some others start with "A specific...", while others start with "Specifies...". I wanted to make sure this was intentional?

Reply:
Good catch. Not intentional....for the others, I copy/pasted the definition of the same flag from saucectl, since it was already formatted, but for these proxy ones, I used the description that was already here. I don't have a strong opinion about this -- whichever you prefer is fine with me -- but you're right; we should be consistent. Let me know what you want.

Reply:
Thank you for clarifying! I think keeping "Specifies..." is good because "A specific..." only appears a handful of times
<img width="736" alt="Screen Shot 2022-02-23 at 7 45 33 PM" src="https://user-images.githubusercontent.com/56411016/155454429-559734b2-e675-432b-8605-edee6b60b3d9.png">
.